.eslintignore

```diff
@@ -12,6 +12,7 @@
 
 # misc
 /coverage/
+/utils/
```

Comment:
This files should be linted, change the configuration for this directory from `node` to `browser`

Reply:
Finally, the file is included in `lib/utils` as the `customElements.define` script will be always included in the bundle.
Changed in d6373b6